include/spirv-tools/libspirv.h

```diff
@@ -366,6 +370,18 @@ typedef enum {
   SPV_ENV_OPENGL_4_5,  // OpenGL 4.5 plus GL_ARB_gl_spirv, latest revisions.
 } spv_target_env;
 
+// SPIR-V Validator can be parameterized with the following Universal Limits.
+typedef enum {
+  validator_limit_max_struct_members,
```

Comment:
In C, enumerant names are not scoped.  So please prefix this with `spv_`.

But thanks for making this enum in the first place!